libs/execution_context/ebpf_maps.c

```diff
@@ -203,9 +211,10 @@ typedef struct _ebpf_map_function_table
         _In_ ebpf_core_map_t* map, _In_ const uint8_t* key, _In_ const uint8_t* value, ebpf_map_option_t option);
     ebpf_result_t (*delete_entry)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* key);
     ebpf_result_t (*next_key)(_In_ ebpf_core_map_t* map, _In_ const uint8_t* previous_key, _Out_ uint8_t* next_key);
-} ebpf_map_function_table_t;
+    uint64_t flags;
```

Comment:
bitfields are much easier to debug than flags.
```suggestion
    int zero_length_key : 1;
    int zero_length_value : 1;
    int per_cpu : 1;
    int key_history : 1;
```